shell/list/fleet.cattle.io.bundle.vue

```diff
@@ -3,6 +3,7 @@ import { FLEET } from '@shell/config/types';
 import { Banner } from '@components/Banner';
 import Loading from '@shell/components/Loading';
 import ResourceTable from '@shell/components/ResourceTable';
+import ResourceFetch from '@shell/mixins/resource-fetch';
```

Comment:
I'm not sure why, but in the fleet product for all resource types hitting the refresh button causes the `loading` indicator to show until all incremental requests have completed (easiest to test with the [example repo](https://github.com/rancher/fleet-examples), on the bundles resource list with a throttled network). 

This correctly doesn't happen in other resources, such as Storage --> Secrets

Update: This also seems the case with the CIS scan resources

Update 2: And Apps --> Installed Apps

Reply:
I think I have this figured out: 
- There is a quite substancial number of custom list view that rely on `<Loading v-if="$fetchState.pending" />` with a v-else to display the list. Since we haven't "finished" the requests we are technically still `pending`
- This doesn't happen on `Secrets` because it doesn't have a custom list view. It relies on the default list view (`ResourceList` entrypoint) which does things a bit different. There we don't employ a global `<Loading>` but use a local loading state on the `ResourceTable` component with a computed that derives from this condition:
`return this.hasData ? false : this.$fetchState.pending;`

I'll have to change 20 files in order to fix this on the custom list views. Is it a mandatory requirement for this feature? I agree that it would be nice to have this done and that if it's the only issue left from your comments, maybe someone can give the final approval later. What do you think?

Reply:
changed to use `loading` prop on `ResourceTable` instead of `<Loading>` component